storage/validateArtifactLowerCase/README.MD

```diff
@@ -0,0 +1,13 @@
+Artifacts Upload Validation Checks for lowercase Artifact Upload.
+=====================================
+##Description:
+This plugin helps you validate upload to artifact based on your condition in this case its a lowercase artifact name example. 
+It can be run automatically as a scheduled job.
```

Comment:
This is inaccurate, there is no scheduled job. The plugin runs automatically on artifact deployment.